app/models/event.rb

```diff
@@ -1,14 +1,6 @@
 class Event < ApplicationRecord
-	#Allowing URL, end_date, and address2 to be empty 
-	validates :name, presence: true
-	validates :description, presence: true
-	validates :start_date, presence: true
-	validates :address1, presence: true
-	validates :city, presence: true
-	validates :state, presence: true, length: {is: 2} #Dropdown selection on front end?  
-	VALID_ZIP_REGEX= /\A\d{5}-?\d{4}?\z/
-	validates :zip, presence: true, format:{with:VALID_ZIP_REGEX}
-	validates :scholarship_available, inclusion: {in: [true, false]}
-	VALID_URL_REGEX = URI::regexp 
+	validates :name, :start_date, :source_updated, :source_type, :address1, :city, presence: true
```

Comment:
Since new columns are being added, the existing ones will not have data in those newly validated columns.  Which could cause an issue.

Typically the migration would also deal with resolving these issues.  But perhaps in this case we should just remove any validation of `presence` for any newly created columns.

Reply:
There shouldn't be any existing data.  I wrote the event.rb endpoint, and when it was originally created the plan was to allow operation code community leaders to add data on the front end.  This was never implemented, and that's when we decided to get data from Meetup and not allow events to be manually entered.  So we should be starting fresh with event data once this is merged.

Reply:
Gotcha.  Can you please confirm with @hollomancer that `Event.count == 0` in prod.  If yes, we're set on the validations.

Reply:
Confirmed!